src/hooks/useStablecoinPrice.ts

```diff
@@ -28,11 +27,7 @@ export default function useStablecoinPrice(currency?: Currency): Price<Currency,
   const amountOut = chainId ? STABLECOIN_AMOUNT_OUT[chainId] : undefined
   const stablecoin = amountOut?.currency
 
-  // TODO(#2808): remove dependency on useBestV2Trade
-  const v2USDCTrade = useBestV2Trade(TradeType.EXACT_OUTPUT, amountOut, currency, {
-    maxHops: 2,
-  })
-  const v3USDCTrade = useClientSideV3Trade(TradeType.EXACT_OUTPUT, amountOut, currency)
+  const trade = useRoutingAPITrade(TradeType.EXACT_OUTPUT, amountOut, currency, RouterPreference.CLIENT)
```

Comment:
Does this break client-side-only price impact calculations?

Reply:
why would it do that? this uses the client router so it should only do client calculations. pote didn't want to add more load to the routing api

Reply:
lol I missed the v obvious `RouterPreference.CLIENT)` and just saw the fn name change